crAPI/openresty_nginx.conf

```diff
@@ -0,0 +1,91 @@
+# nginx.conf  --  docker-openresty
```

Comment:
This file should not be checked in. The crAPI/services/web/nginx.conf.template should be modified instead, and the template will get copied at build time.

If this file gets checked in, local development will break.

Reply:
@hnataraj This isn't the actual Nginx config file but the OpenResty config file which internally picks the Nginx config files. I needed this in crapi-all, mainly because we're installing OpenResty ourselves in this image whereas the web service Docker image is based on top of the OpenResty image. This file should already be in the base image in the web service so shouldn't impact the local dev env.

Ideally, we should only touch this if we want to change any OpenResty stuff, not our Nginx conf. Makes sense?

Reply:
Which means if I need to make a change in routes, I need to remember to update both conf files, lest the monolith will break.

is that correct?

Reply:
No, you just make changes in the service nginx conf file. We're not putting anything related to the crAPI web service here.